odkx-src/basics-install.rst

```diff
@@ -10,14 +10,17 @@ Prerequisites
 
 You must have an Android tablet with an operating system version 4.4 or higher.
 
+If you are working on a Windows/Mac/Linux machine, you can use `Android Studio <https://developer.android.com/studio>` to launch an Android emulator.
```

Comment:
Please clarify that you can use the emulator for testing purposes (not data collection).

Reply:
@elmps2018 surprisingly people do use the emulator for data collection. It works, I wouldn't but they do.